packages/fxa-support-panel/config/index.ts

```diff
@@ -66,6 +66,16 @@ const conf = convict({
         format: ['default_fxa', 'dev_fxa', 'default', 'dev', 'short', 'tiny']
       }
     }
+  },
+  security: {
+    csp: {
+      frameAncestors: {
+        default: 'none',
+        doc:
+          'https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors',
+        env: 'CSP_FRAME_ANCESTORS'
```

Comment:
@jrgm Here's the new env var.

Reply:
Thanks @chenba 

@marty331 can you let me know what the origin domains should be in dev, stage and prod?

Reply:
> @marty331 can you let me know what the origin domains should be in dev, stage and prod?

Or if more then one, or a wildcard pattern, what that is.

Reply:
@jrgm for Zendesk right?

Reply:
@marty331 - yes, for the zendesk domain(s) that frame the support-panel

Reply:
It should always be `https://mozilladev.zendesk.com`